### DISCUSSION TITLE
Unable to create variant in Prompt Flow VSCode

### DISCUSSION THREAD

**Discussion by sargupta** [microsoft/promptflow#2143] [Q&A] [answered]
I have been trying to create a variant from the LLM tool, though I am able to do so in Azure ML PF UI however, when trying the same in VS Code, it's giving the following error: 
![image (1)](https://github.com/microsoft/promptflow/assets/28517335/4b3d2dcb-bfd0-4100-ae04-b9371129a105)
![image (2)](https://github.com/microsoft/promptflow/assets/28517335/5ef8c9a1-2208-4d80-81f2-7419219b679a)

basically, it's not recognizing the  jinja2 file of the variant.
Note: this is irrespective if I am using same connection or a different one

Any thoughts?

**Comment by 0mza987** [comment, marked as answer]
Tracked in https://github.com/microsoft/promptflow/issues/2144
